src/realmCryo.ttl

```diff
@@ -234,24 +431,59 @@ soreac:SeaIce rdf:type owl:Class ;
+            skos:definition  [
+              rdfs:comment  "Water ice which has formed by the freezing of sea water."@en ;
+              dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+              dcterms:created "2020-02-12T13:43:39-08:00Z"^^xsd:dateTimeStamp ;
+              dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+              prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_00002200> ;
+            ] .
 
 
 ###  http://sweetontology.net/realmCryo/SeasonalIce
 soreac:SeasonalIce rdf:type owl:Class ;
                  rdfs:subClassOf soreac:SeaIce ;
-                 rdfs:label "seasonal ice"@en .
+                 rdfs:label "seasonal ice"@en ;
+                 skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_03000071> ;
+                 rdfs:comment "Sea ice develops from young ice; thickness from 0.3 to 2 meters (1 to 6.6 feet).  Sea ice be subdivided into thin first-year ice (white ice), medium first-year ice, and thick first-year ice.  First-year ice is distinguished from older ice primarily by having a higher salinity. Undeformed first-year ice differs from older ice in that it is smoother and lacks refrozen melt ponds. Characteristically level where undisturbed by pressure, but where ridges occur, they  distinguished by being larger, more angular, and more porous than multiyear ridges."@en ;
+                 skos:definition  [
+                   rdfs:comment  "Sea ice which has formed over a single freezing season."@en ;
+                   dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+                   dcterms:created "2020-02-12T13:45:28-08:00Z"^^xsd:dateTimeStamp ;
+                   dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                   prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_03000071> ;
+                 ] .
 
 
 ###  http://sweetontology.net/realmCryo/Snout
 soreac:Snout rdf:type owl:Class ;
-           rdfs:label "snout"@en .
+           rdfs:label "snout"@en ;
+           skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001529> ;
+           skos:definition  [
+             rdfs:comment  "An ice mass which constitutes that part of a glacier which has the lowest elevation."@en ;
```

Comment:
same definition as glacial terminus... See comment there and clarify distinction between these.

Reply:
probably another ENVO definition out of context.  Part of broader problem of how/if term in SWEET should be defined.